### ISSUE TITLE
Multiple layers of armour

### ISSUE THREAD

**Issue by OctarineSourcerer** [OctarineSourcerer/cyberpunk2020-foundry#25]
Characters can have multiple layers of armor, have this doable? I'm thinking a second combat or gear tab for arranging it, with some drag-and-drop. This will, in any case, involve some armor errata in the second edition.

This will not be in the initial release. For now, will just use armour with the greatest SP.

**Comment by drowzero0** [comment]
at the moment it ADDS UP every armour given.
it does not take the highest value.
![image](https://user-images.githubusercontent.com/62395017/114847920-7ff51200-9dde-11eb-906c-2108c9e3806d.png)

**Comment by OctarineSourcerer** [comment]
Okay, so just checked the rulebook for this one - I think I'd had adding them together as the most simple early implementation, and completely forgotten the idea of using the armor with the greatest SP.
That being said, I'll likely end up having an option in the system settings: "Use Proportional Armor". If off, SPs will add together - I think that's more intuitive to players than just using the greatest. If on, it will use the Proportional Armor rule in the book:
![image](https://user-images.githubusercontent.com/6842867/114863217-5ee4ef00-9de7-11eb-82a9-b154f6135db1.png)
![image](https://user-images.githubusercontent.com/6842867/114863230-64dad000-9de7-11eb-967f-d5cdfaa5ff62.png)

How does that sound?

**Comment by drowzero0** [comment]
sounds good to me. especially the "proportional Armor Table" makes it more clear to me.
the layering and the decrease of REF going in with this is ok too, but then you have to check doubled for stats. 
1st if there is a Defence stat already on a armor-zone and creating a 2nd layer to add a - REF 
2nd does the item itself already have a reduction on REF to sum them up correctly.

for example 
a leather shirt and a kevlar shirt both have a 0 -REF change, but together its -1 REF
but adding a kevlar shirt (0) and a heavy kevlar jacket (-3) makes it together -4 REF

**Comment by OctarineSourcerer** [comment]
> sounds good to me. especially the "proportional Armor Table" makes it more clear to me.
> the layering and the decrease of REF going in with this is ok too, but then you have to check doubled for stats.
> 1st if there is a Defence stat already on a armor-zone and creating a 2nd layer to add a - REF
> 2nd does the item itself already have a reduction on REF to sum them up correctly.
> 
> for example
> a leather shirt and a kevlar shirt both have a 0 -REF change, but together its -1 REF
> but adding a kevlar shirt (0) and a heavy kevlar jacket (-3) makes it together -4 REF

Yes, that checking would have to be done. It's totally reasonably possible - I can distinguish the two penalties to REF to be `encumbrancePenalty` and `layeringPenalty`. The first just sums up encumbrance values, the second would be layering. Total REF would take both of those into account.

I would, at that point, _definitely_ like to at least have some clearer indication of stat modifiers before going ahead with this, the reason being that at that point there's a fair bit of maths going on in the background that wouldn't be clear to players at all. So yes, I will have these rules in, but only after I've figured out a sensible way to present stats being modified to the user. I'll modify one of the earlier issues (stats indicating wound effect) to be more generic for this purposes.

**Comment by drowzero0** [comment]
yep it will be a bit more math than just adding a stat to a skill.
guess creating a visual for new players will be the hardest part, if you did not read the rulebook carefully before.
but i guess the 2 classes encumbrancePenalty and layeringPenalty are a good way to make it clear

the statmodifier seem to be endless, just to name another possible modifier which can happen later on:
![image](https://user-images.githubusercontent.com/62395017/114866471-e6385f80-9df3-11eb-94d6-105924f01b7b.png)
maybe i can find a list which has every possible modifier for stats, otherwise i can create one

**Comment by OctarineSourcerer** [comment]
> yep it will be a bit more math than just adding a stat to a skill.
> guess creating a visual for new players will be the hardest part, if you did not read the rulebook carefully before.
> but i guess the 2 classes encumbrancePenalty and layeringPenalty are a good way to make it clear
> 
> the statmodifier seem to be endless, just to name another possible modifier which can happen later on:
> ![image](https://user-images.githubusercontent.com/62395017/114866471-e6385f80-9df3-11eb-94d6-105924f01b7b.png)
> maybe i can find a list which has every possible modifier for stats, otherwise i can create one

Making a full list shouldn't be needed - I'll essentially slowly accumulate more and more modifiers as the system gets more complete. I'll likely look into Foundry's `ActiveEffects` for something like this, or just have an array of modifiers and their text under the hood. That second option is all too likely - ActiveEffects doesn't seem to have much documentation at all at the moment, and I'll have to do some digging in people's systems and the Discord to figure out how it works.
